jujugui/static/gui/src/app/components/create-payment-user/create-payment-user.js

```diff
@@ -15,7 +15,7 @@ class CreatePaymentUser extends React.Component {
     this.xhrs = [];
     this.state = {
       billingAddressSame: true,
-      business: false,
```

Comment:
I'd prefer if this was a more generic 'state' so that it wasn't simply a boolean value so we could have more versions.. but that can probably be done as a follow-up if we run into that situation.

`{ displayType: 'business' }`